tmt/steps/discover/fmf.py

```diff
@@ -118,7 +120,7 @@ class DiscoverFmf(tmt.steps.discover.DiscoverPlugin):
         )
 
     @classmethod
-    def options(cls, how=None):
+    def options(cls, how: Optional[str] = None) -> Any:
```

Comment:
Should return `List[ClickOptionDecoratorType]` which is implemented in #1304.

Reply:
thanks! wasn't aware of that object